tests/test_elastic.py

```diff
@@ -397,6 +397,18 @@ def test_unreg_users_dont_show_in_search(self):
         contribs = search.search_contributor(unreg.fullname)
         assert_equal(len(contribs['users']), 0)
 
+
+    def test_unreg_users_do_show_on_projects(self):
```

Comment:
Yeah, I think that test is already in there (right above the one I added, though you can't see it in the diff)